src/DependencyInjection/CsaGuzzleExtension.php

```diff
@@ -123,4 +119,16 @@ private function processClientsConfiguration(array $config, ContainerBuilder $co
             }
         }
     }
+
+    private function findSubscriberIds(array $explicitlyConfiguredIds, ContainerBuilder $container)
+    {
+        $allNames = array_values(array_map(
+            function ($tags) { return $tags[0]['alias']; },
+            $container->findTaggedServiceIds(SubscriberPass::SUBSCRIBER_TAG)
+        ));
+
+        return array_filter($allNames, function ($name) use ($explicitlyConfiguredIds) {
+            return !isset($explicitlyConfiguredIds[$name]) || $explicitlyConfiguredIds[$name];
+        });
+    }
```

Comment:
@csarrazi alright, this *should* work now in a BC-compatible way (almost). The only problem is that any subscribers configured in subsequently booted bundles will not be added to clients added via configuration.

```php
$bundles[] = new Csa\Bundle\GuzzleBundle\CsaGuzzleBundle();
$bundles[] = new This\Bundle\Adds\SomeMoreSubscribers(); // <-- none will be added from this bundle
```

This was not a problem before, because all this was done in the factory at runtime. Not really sure what to do about it though. Do you have any ideas? :)